src/components/common/FormButton.js

```diff
@@ -414,7 +420,7 @@ const FormButton = ({
         disabled={disabled}
         onClick={(e) => {
             onClick && onClick(e)
-            linkTo && history.push(linkTo)
+            linkTo && (linkTo.includes('http') ? window.location = linkTo : history.push(linkTo))
```

Comment:
not sure I understand logic here

is the idea to check whether this is full URL or relative URL? If so – better to check it with more specific check, e.g. https://stackoverflow.com/a/19709846

as otherwise somebody might have hard time if they have `http` elsewhere in the URL :)